onchain/rollups/contracts/library/LibOutputValidation.sol

```diff
@@ -105,7 +190,7 @@ library LibOutputValidation {
         // is contained in it. We can't simply use hashOfOutput because the
         // log2size of the leaf is three (8 bytes) not  five (32 bytes)
         bytes32 merkleRootOfHashOfOutput = MerkleV2.getMerkleRootFromBytes(
-            abi.encodePacked(keccak256(encodedOutput)),
+            abi.encodePacked(keccak256(abi.encode(output))),
```

Comment:
**Note:** this is a temporary solution to generate correct output validity proofs. We would like to hash `output` directly, without having to ABI-encode it.